### ISSUE TITLE
Main Menu Content are not usable by keyboard and screen readers. 

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/pul_library_drupal#1823]
The main menu flyouts cannot be operated by keyboard and screen reader users, and the alternate path to the content (clicking the top level link to go to a landing page) does not have all the content of the flyout.

Proposed Solution:

```Either expose subitems on :focus, or separate the downward arrow into a button to create a clickable disclosure menu.```
